### ISSUE TITLE
Add fields for M-22-18 requirements in the IT standards report (attestation letter yes/no, 

### ISSUE THREAD

**Issue by hatfieldjm4** [GSA/GEAR3#140]
Use case: contractor needs to know if a software has an attestation letter.
- Field that says the "requires attestation letter" (yes/no) – globally default to “no” for all old technologies.
- Field for yes/no for FedRAMP'd
- Field, similar to “related artifacts” in FISMA inventory, they need a link. 
- Field for yes/no for open source (lesser importance)
- Add definitions for these new fields in the data dictionary and the API documentation 

We can reuse the "Available through MyView" and "Gold Image" fields since these are yes/no fields, are already developed out in GEAR manager and in the database, and have been indicated by Cindy that they are currently not needed.

**Comment by hatfieldjm4** [comment]
@kinggsagov I've added 4 data fields (columns) to obj_technology in the dev database (gear_schema): 

1. attestation_required - on the front end, this should read "Attestation Required"
2. attestation_link -  on the front end, this should read "Attestation Document"
3. fedramp -  on the front end, this should read "FedRAMP"
4. open_source - on the front end, this should read "Open Source"

As mentioned in our meeting, these will need to be added to the API and then added to:
1. The report view (i.e. the report you see when you click on Technologies --> IT Standards List)
2. The modal view (the view that appears when you click into an individual software listing on the IT standards list report)
3. The GEAR Manager user form (the form that appears when you click "edit this item" at the bottom right of the modal view)

**Comment by hatfieldjm4** [comment]
@kinggsagov Hi King-- I pulled the changes down to dev.ea.gsa.gov, and it looks like the GEAR Manager form is not saving the following fields properly: 
- FedRAMP true/false, 
- Attestation letter true/false, 
- Attestation letter URL, and 
- Open source true/false fields

When you save them, they do not appear on the modal or the report views, but when you go back into the GEAR Manager form for that specific technology, the changes do appear. It does appear to be saving to the database properly. 

Also, I noticed an error that is occurring when editing technologies in the IT standards report that have both a software end-of-life date and an approval expiration date. I'll create a separate ticket for this issue.
